tests/test_pipeline.py

```diff
@@ -663,28 +663,39 @@ def test_13_03_save_long_measurements(self):
 
     def test_14_01_unicode_save(self):
         pipeline = get_empty_pipeline()
-        module = MyClassForTest0801()
+        module = TestModuleWithMeasurement()
+        # Little endian utf-16 encoding
         module.my_variable.value = u"\\\u2211"
+        module.other_variable.value = u"\u2222\u0038"
         module.module_num = 1
         module.notes = u"\u03B1\\\u03B2"
         pipeline.add_module(module)
         fd = cStringIO.StringIO()
         pipeline.savetxt(fd, save_image_plane_details=False)
         result = fd.getvalue()
         lines = result.split("\n")
-        self.assertEqual(len(lines), 10)
+        self.assertEqual(len(lines), 11)
+        text, value = lines[-3].split(":")
+        #
+        # unicode encoding:
+        #     backslash: \\ (BOM encoding)
+        #     unicode character: \u
```

Comment:
I think you mean \u2211 here.  BTW, that's the n-ary summation character, if you want to name it (as you did for the backslash above).